tests/argparse/test_argparse.py

```diff
@@ -181,6 +181,44 @@ def test_add_argument_url(self) -> None:
         args = parser.parse_args(cmd_args)
         assert args.url == make_url("https://github.com")
 
+    @pytest.mark.dependency(depends=["add_argument"])
+    @pytest.mark.parametrize(
+        "value, value_type, min_value, max_value, expectation",
+        [
+            param("3", int, None, None, does_not_raise(), id="Value has expected type"),
```

Comment:
```suggestion
            param("3", int, None, None, does_not_raise(), id="Value has expected type"),
            param("3", int, 3, 3, does_not_raise(), id="Value has expected type"),
```

Reply:
Good point, although not so much unit testing (it is more about functionality, so this does not increase the code coverage), I like the idea of testing this. Will add my own suggestion though to change the ID description:
```
param("3", int, 3, 3, does_not_raise(), id="Value equal to minimum and maximum values"),
```